packages/cli/src/__tests__/commands/__snapshots__/Init.test.ts.snap

```diff
@@ -1,7 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`appends when .env present 1`] = `
-Environment variables loaded from .env
```

Comment:
Is this test not loading from env anymore?

Reply:
Indeed see description of the PR for the list of commands not loading .env anymore

Reply:
Do you think it's needed? I checked quick but I didn't see how it would be used there, feel free to double check 👍🏼